ts/screens/wallet/ConfirmCardDetailsScreen.tsx

```diff
@@ -252,29 +258,41 @@ class ConfirmCardDetailsScreen extends React.Component<Props, State> {
             hideFavoriteIcon={true}
           />
           <View spacer={true} />
-          <NoticeBox
-            backgroundColor={customVariables.toastColor}
-            iconProps={{
-              name: "io-notice",
-              color: customVariables.brandDarkGray,
-              size: 24
+          <InfoBox
+            alignedCentral={true}
+            iconSize={24}
+            iconColor={IOColors.bluegreyDark}
+          >
+            <H5 weight={"Regular"}>{I18n.t("wallet.saveCard.notice")}</H5>
+          </InfoBox>
+          <View spacer large />
+          <View
+            style={{
+              flexDirection: "row",
+              justifyContent: "space-between"
             }}
+            testID={"test"}
           >
-            <Text>{I18n.t("wallet.saveCard.notice")}</Text>
-          </NoticeBox>
-          <View spacer={true} />
-          <Grid>
-            <Col size={5}>
-              <Text bold={true}>{I18n.t("wallet.saveCard.infoTitle")}</Text>
-              <Text>{I18n.t("wallet.saveCard.info")}</Text>
-            </Col>
-            <Col size={1}>
+            <View
+              style={{
+                ...IOStyles.flex
+              }}
```

Comment:
```suggestion
              style={IOStyles.flex}
```

Reply:
Thank you. Refactored in [25a478f](https://github.com/pagopa/io-app/pull/3691/commits/25a478f07dc3eca4cf83e880b01d3de1774807d5)